server ORM/tests/artist.test.js

```diff
@@ -0,0 +1,60 @@
+const request = require("supertest");
+const app = require("../app");
+const { Artist } = require("../models");
+
+const artistMock = [
+    {
+        id: 1,
+        name: "Dor",
+        coverImg: "img1",
+        releasedAt: "2020-09-12",
+    },
+    {
+        id: 2,
+        name: "Kachlon",
+        coverImg: "img2",
+        releasedAt: "2020-09-12",
+    },
+];
+
+describe("api artist", () => {
+    beforeEach(async () => {
+        await Artist.destroy({ truncate: true, force: true });
+    });
+
+    it("Can create artists and get them", async () => {
+        await request(app).post("/api/artists").send(artistMock[0]);
+        await request(app).post("/api/artists").send(artistMock[1]);
+        const { body } = await request(app).get("/api/artists");
+        expect(body[0].name).toBe(artistMock[0].name);
+        expect(body[1].name).toBe(artistMock[1].name);
+    });
```

Comment:
Better not to relay on the order that object will return from API - Good in case the API support some order, here it’s just return all “artists”